### ISSUE TITLE
  Chinese users are not responding to installation or search commands. Should it be the source?

### ISSUE THREAD

**Issue by zjarlin** [microsoft/winget-cli#1126]
# Brief description of your issue

# Steps to reproduce

Chinese users are not responding to installation or search commands. Should it be the source?

# Expected behavior

# Actual behavior

# Environment

```none
[winget --info]
Windows Package Manager version
Windows: Windows.Desktop version
Package: Microsoft.DesktopAppInstaller version

Any other software?
```

**Comment by denelon** [comment]
@zjarlin I believe translation failed.

If you search for packages using Chinese characters, and a manifest provides a locale file with the Chinese data, you should see results.  I believe Microsoft.WindowsTerminal has examples for both traditional Chinese and simplified Chinese.

**Comment by denelon** [comment]
@zjarlin

We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) and updated our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting).

Can you confirm if this is still happening?
